controllers/api.js

```diff
@@ -296,45 +301,49 @@ exports.postTwitter = (req, res, next) => {
 exports.getSteam = (req, res, next) => {
   const steamId = '76561197982488301';
   const params = { l: 'english', steamid: steamId, key: process.env.STEAM_KEY };
-  async.parallel({
-    playerAchievements: (done) => {
+  const fetch = promisify(request.get);
```

Comment:
DONE. I removed auxiliary `fetch` variable and promisified whole `request` library:

```js
const request = bluebird.promisifyAll(require('request'), { multiArgs: true });
```